src/ansys/grantami/bomanalytics/queries.py

```diff
@@ -514,7 +535,11 @@ def _call_api(self, api_method: Callable[..., models.ModelBase], arguments: Dict
     @abstractmethod
     def _run_query(
         self,
-        api_instance: Union[api.ComplianceApi, api.ImpactedSubstancesApi, api.SustainabilityApi],
+        api_instance: Union[  # type: ignore[override]
```

Comment:
I couldn't figure out how to make a type variable for this method with the right contravariant property, so I just ignored the warning